docs/tutorials/exploring_bdk_rn.md

```diff
@@ -0,0 +1,655 @@
+---
+title: "BDK-RN: Building React Native Apps with BDK"
+description: "A guide to using bdk-rn for building bitcoin apps"
+authors:
+    - Bitcoin Zavior
+date: "2022-08-05"
+tags: ["bdk-rn", "bdk", "tutorial", "guide", "wallet"]
+---
+
+## Introduction
+
+The **BitcoinDevkit**'s **React Native** library, bdk-rn enables building bitcoin applications for Android and iOS mobile platforms. Using bdk-rn does not require knowledge of the underlying bitcoin or BDK API, and using bdk-rn is similar to using any other RN module. 
+
+In this tutorial, we will explore bdk-rn usage and the API it provides. This guide will walk through the development process and code for making a bitcoin application. The bitcoin application we create will be a non-custodial HD Wallet. The application will have the functionality to create a new wallet or restore from a known mnemonic seed phrase. The application will also be able to interact with the bitcoin network to sync UTXOs from new blocks and broadcast transactions.
+
+The tutorial will focus on bitcoin and bdk-rn concepts and API. So it will gloss over react native related aspects. The code for this tutorial is available on GitHub at https://github.com/LtbLightning/AwesomeBitcoinApp
```

Comment:
```suggestion
The tutorial will focus on bitcoin and bdk-rn concepts and API. So it will gloss over react native related aspects. The code for this tutorial is available on the [LtbLightning GitHub](https://github.com/LtbLightning/AwesomeBitcoinApp)
```

Reply:
Done!